.env.example

```
@@ -1,40 +1,32 @@
-# Leave blank to disable Google Calendar sync
-# To enable:
-# 1. Go to https://console.cloud.google.com/
-# 2. Create a service account
-# 3. Download the JSON key file
-# 4. Copy the ENTIRE JSON content as a single line below
-# Example: GOOGLE_CREDENTIALS={"type":"service_account","project_id":"my-project",...}
-GOOGLE_CREDENTIALS=
+# Google Calendar Integration (Optional)
+# Service account JSON credentials
+# Get from: https://console.cloud.google.com/iam-admin/serviceaccounts
+GOOGLE_CREDENTIALS={"type":"service_account","project_id":"your-project","private_key_id":"...","private_key":"-----BEGIN PRIVATE KEY-----\n...\n-----END PRIVATE KEY-----\n","client_email":"your-service-account@your-project.iam.gserviceaccount.com","client_id":"...","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_x509_cert_url":"..."}
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)